docs/src/content/docs/getting-started/concepts.mdx

```diff
@@ -29,34 +34,72 @@ Repository
+when_modified:
+  file_patterns:
+    - '**/*.tf'
+    - '**/*.tfvars'
+```
 
-Terrateam maps the detected changes to their appropriate [Dirspaces](/getting-started/concepts#dirspace). These mapped Dirspaces are referred to as Changes. When Terrateam executes a Plan or Apply operation on the changes in a pull request, it only considers the Dirspaces that have files modified in the pull request.
+This configuration tells Terrateam to monitor changes to any files with `.tf` or `.tfvars` extensions. Modify these patterns in your configuration file to match your project's structure.
 
-## Auto-Plan and Auto-Apply
+Terrateam maps detected changes to their appropriate [Dirspaces](/getting-started/concepts#dirspace). These mapped Dirspaces are referred to as Changes. When executing operations, Terrateam only considers Dirspaces with modified files in the pull request.
 
-Terrateam offers two automated operations:
+## Auto-Plan and Auto-Apply
 
-- **Auto-Plan**: Automatically runs a Plan on a new pull request or an update to an existing one. This allows you to see the proposed changes to your infrastructure before merging the pull request.
-- **Auto-Apply**: Automatically runs an Apply after merging a pull request. This ensures that the approved changes are applied to your infrastructure immediately after the merge.
+Terrateam offers two configurable automated operations:New chat
+  auto_plan: true  # Set to false to disable
+  ```
+- **Auto-Apply**: (Disabled by default) Runs `terraform apply` post-merge. Enable with:
+  ```yaml
+  auto_apply: true
+  ```
 
-These automated operations streamline your workflow and reduce the manual effort required to manage your infrastructure.
+These automated operations streamline workflows while maintaining control through configurable thresholds.
 
 ## Locks and Unlocks
 
-Terrateam Locks make sure that only a single change to a resource can be applied in a pull request at any given time. This prevents conflicts and maintains consistency in your infrastructure.
+Terrateam's locking system prevents concurrent modifications by ensuring only one change can be applied to a resource at a time. This prevents:
 
-* **Lock**: A guarantee that only a single change to a resource can be applied in a pull request at any given time.
-* **Unlock**: The process of releasing a lock, allowing other changes to be applied.
+- State file conflicts  
+- Resource modification races  
+- Concurrent apply operations  
 
+**Key behaviors**:
+- Enabled by default (no config needed)
+- Automatically released after apply + merge
+- Force unlock via comment: `terrateam unlock`
+
+```yaml
+# .terrateam/config.yml (default shown)
+workflows:
+  lock_policy: strict  # strict|apply|merge|none
+```
+
+For advanced configurations see our [Locks & Concurrency guide](/advanced-workflows/locks-and-concurrency).
+
+:::note
+ Default settings work for most teams. Customize only if you need specific lock timing or policies.
+:::
 ## Tags and Tag Sets
```

Comment:
Tags and tag queries are introduced a little late. Moving them higher in the document, preferably before locks, would be a good idea.